src/bramble/client/main.js

```diff
@@ -417,6 +421,7 @@ define([
                                     secondPaneWidth: _state.secondPaneWidth,
                                     previewMode: _state.previewMode,
                                     wordWrap: _state.wordWrap,
+									closeTags: _state.closeTags,
```

Comment:
Oh, I'm aware there are many indenting inconsistencies, I was planning to do that after I ever solve this.